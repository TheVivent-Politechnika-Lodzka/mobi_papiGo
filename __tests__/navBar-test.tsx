import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import NavBar from '../src/components/navBar';


it('Bar test', () => {
  const bar = renderer.create(<NavBar navigation={4} user='mobilki'/>).toJSON();
  expect(bar).toMatchSnapshot();
});