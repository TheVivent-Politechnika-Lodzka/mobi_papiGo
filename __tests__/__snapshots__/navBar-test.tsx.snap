// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar test 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#87AAAA",
      "bottom": 0,
      "flexDirection": "row",
      "height": "12%",
      "justifyContent": "space-around",
      "left": 0,
      "padding": "2%",
      "position": "absolute",
      "right": 0,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "borderRadius": 3,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 3,
        },
        Object {
          "height": "100%",
          "width": "20%",
        },
        false,
      ]
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "busy": false,
          "disabled": false,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#C8E3D4",
            "borderColor": "#2089dc",
            "borderRadius": 3,
            "borderWidth": 0,
            "flexDirection": "row",
            "height": "100%",
            "justifyContent": "center",
            "padding": 8,
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Object {
              "backgroundColor": "transparent",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <Image
            onLoad={[Function]}
            source={
              Object {
                "testUri": "../../../assets/lapa.png",
              }
            }
            style={
              Object {
                "bottom": 0,
                "height": 50,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 50,
              }
            }
            testID="RNE__Image"
            theme={
              Object {
                "colors": Object {
                  "black": "#242424",
                  "disabled": "hsl(208, 8%, 90%)",
                  "divider": "#bcbbc1",
                  "error": "#ff190c",
                  "grey0": "#393e42",
                  "grey1": "#43484d",
                  "grey2": "#5e6977",
                  "grey3": "#86939e",
                  "grey4": "#bdc6cf",
                  "grey5": "#e1e8ee",
                  "greyOutline": "#bbb",
                  "platform": Object {
                    "android": Object {
                      "error": "#f44336",
                      "grey": "rgba(0, 0, 0, 0.54)",
                      "primary": "#2196f3",
                      "searchBg": "#dcdce1",
                      "secondary": "#9C27B0",
                      "success": "#4caf50",
                      "warning": "#ffeb3b",
                    },
                    "default": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "ios": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "web": Object {
                      "error": "#ff190c",
                      "grey": "#393e42",
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#ca71eb",
                      "success": "#52c41a",
                      "warning": "#faad14",
                    },
                  },
                  "primary": "#2089dc",
                  "searchBg": "#303337",
                  "secondary": "#ca71eb",
                  "success": "#52c41a",
                  "warning": "#faad14",
                  "white": "#ffffff",
                },
              }
            }
            transition={true}
            transitionDuration={360}
          />
          <View
            accessibilityElementsHidden={true}
            collapsable={false}
            importantForAccessibility="no-hide-descendants"
            nativeID="animatedComponent"
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#bdbdbd",
                  "height": 50,
                  "justifyContent": "center",
                  "width": 50,
                }
              }
              testID="RNE__Image__placeholder"
            />
          </View>
          <View
            style={
              Object {
                "height": 50,
                "width": 50,
              }
            }
            testID="RNE__Image__children__container"
          />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "borderRadius": 3,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 3,
        },
        Object {
          "height": "100%",
          "width": "20%",
        },
        false,
      ]
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "busy": false,
          "disabled": false,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#C8E3D4",
            "borderColor": "#2089dc",
            "borderRadius": 3,
            "borderWidth": 0,
            "flexDirection": "row",
            "height": "100%",
            "justifyContent": "center",
            "padding": 8,
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Object {
              "backgroundColor": "transparent",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <Image
            onLoad={[Function]}
            source={
              Object {
                "testUri": "../../../assets/logo.png",
              }
            }
            style={
              Object {
                "bottom": 0,
                "height": 50,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 50,
              }
            }
            testID="RNE__Image"
            theme={
              Object {
                "colors": Object {
                  "black": "#242424",
                  "disabled": "hsl(208, 8%, 90%)",
                  "divider": "#bcbbc1",
                  "error": "#ff190c",
                  "grey0": "#393e42",
                  "grey1": "#43484d",
                  "grey2": "#5e6977",
                  "grey3": "#86939e",
                  "grey4": "#bdc6cf",
                  "grey5": "#e1e8ee",
                  "greyOutline": "#bbb",
                  "platform": Object {
                    "android": Object {
                      "error": "#f44336",
                      "grey": "rgba(0, 0, 0, 0.54)",
                      "primary": "#2196f3",
                      "searchBg": "#dcdce1",
                      "secondary": "#9C27B0",
                      "success": "#4caf50",
                      "warning": "#ffeb3b",
                    },
                    "default": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "ios": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "web": Object {
                      "error": "#ff190c",
                      "grey": "#393e42",
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#ca71eb",
                      "success": "#52c41a",
                      "warning": "#faad14",
                    },
                  },
                  "primary": "#2089dc",
                  "searchBg": "#303337",
                  "secondary": "#ca71eb",
                  "success": "#52c41a",
                  "warning": "#faad14",
                  "white": "#ffffff",
                },
              }
            }
            transition={true}
            transitionDuration={360}
          />
          <View
            accessibilityElementsHidden={true}
            collapsable={false}
            importantForAccessibility="no-hide-descendants"
            nativeID="animatedComponent"
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#bdbdbd",
                  "height": 50,
                  "justifyContent": "center",
                  "width": 50,
                }
              }
              testID="RNE__Image__placeholder"
            />
          </View>
          <View
            style={
              Object {
                "height": 50,
                "width": 50,
              }
            }
            testID="RNE__Image__children__container"
          />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "borderRadius": 3,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 3,
        },
        Object {
          "height": "100%",
          "width": "20%",
        },
        false,
      ]
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "busy": false,
          "disabled": false,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#C8E3D4",
            "borderColor": "#2089dc",
            "borderRadius": 3,
            "borderWidth": 0,
            "flexDirection": "row",
            "height": "100%",
            "justifyContent": "center",
            "padding": 8,
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Object {
              "backgroundColor": "transparent",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <Image
            onLoad={[Function]}
            source={
              Object {
                "testUri": "../../../assets/plecak.png",
              }
            }
            style={
              Object {
                "bottom": 0,
                "height": 50,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 50,
              }
            }
            testID="RNE__Image"
            theme={
              Object {
                "colors": Object {
                  "black": "#242424",
                  "disabled": "hsl(208, 8%, 90%)",
                  "divider": "#bcbbc1",
                  "error": "#ff190c",
                  "grey0": "#393e42",
                  "grey1": "#43484d",
                  "grey2": "#5e6977",
                  "grey3": "#86939e",
                  "grey4": "#bdc6cf",
                  "grey5": "#e1e8ee",
                  "greyOutline": "#bbb",
                  "platform": Object {
                    "android": Object {
                      "error": "#f44336",
                      "grey": "rgba(0, 0, 0, 0.54)",
                      "primary": "#2196f3",
                      "searchBg": "#dcdce1",
                      "secondary": "#9C27B0",
                      "success": "#4caf50",
                      "warning": "#ffeb3b",
                    },
                    "default": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "ios": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "web": Object {
                      "error": "#ff190c",
                      "grey": "#393e42",
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#ca71eb",
                      "success": "#52c41a",
                      "warning": "#faad14",
                    },
                  },
                  "primary": "#2089dc",
                  "searchBg": "#303337",
                  "secondary": "#ca71eb",
                  "success": "#52c41a",
                  "warning": "#faad14",
                  "white": "#ffffff",
                },
              }
            }
            transition={true}
            transitionDuration={360}
          />
          <View
            accessibilityElementsHidden={true}
            collapsable={false}
            importantForAccessibility="no-hide-descendants"
            nativeID="animatedComponent"
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#bdbdbd",
                  "height": 50,
                  "justifyContent": "center",
                  "width": 50,
                }
              }
              testID="RNE__Image__placeholder"
            />
          </View>
          <View
            style={
              Object {
                "height": 50,
                "width": 50,
              }
            }
            testID="RNE__Image__children__container"
          />
        </View>
      </View>
    </View>
  </View>
</View>
`;
